lib/numbers.md

```diff
@@ -5,20 +5,45 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What are the different kinds of numbers in Ruby?
 
-Replace me with your response...
+integers (whole numbers), floats (numbers containing decimal points)
 
 # What are some common operations and comparisons you would perform on numbers?
 
-Replace me with your response...
++, *, /, >, <, <=, >=, =, ==, 
 
 # What is the difference between the `+` operation on a number versus on a String?
 
-Replace me with your response...
+With numbers the + opertor will return the total of the two numbers.
 
-# If you have a _String_ `"20"` and want to perform a mathematical operation (like division or multiplication) on it, will it work? If yes, why? If not, how would you make it work?
+1+2 returns 3
 
-Replace me with your response...
+With strings the + operator will return the strings joined together.
+
+"These " + "strings will be joined together" returns "These strings will be joined together"
+
+# If you have a _String_ `"20"` and want to perform a mathematical operation (like division or multiplication) on it, will it work? 
+
+It wont work if you leave "20" as a string. Simply trying to multiply "20"*5 returns 2020202020 (the string "20" five times).
+You need to run the .to_i method on the string first so it treats the string as an interger. "20".to_i*5  returns 100
```

Comment:
How about if I have a string `"20"` and want to divide it by 3? What would you do to the string to prepare it for that mathematical operation? (Assume I want the exact [quotient](https://www.mathsisfun.com/definitions/quotient.html).)